### PR TITLE
Use SafeLayer in Analysis

### PR DESCRIPTION
**Pull request by Gustry**
- Use SafeLayer in Analysis
- Remove keywords when we setup the analysis as the keywords will be read by SafeLayer : 
 - command line
 - dock
 - analysis_handler

### CODE REVIEW

**Comment by Gustry** [comment]
@ismailsunni This PR is now ready to review.
Thanks in advance.

File: safe/storage/safe_layer.py [hunk 1]
```diff
@@ -94,8 +94,8 @@ def keywords(self):
     def keywords(self, keywords):
         """Setter for keywords property.
 
-        :param keywords: The actual layer.
-        :type keywords: QgsMapLayer, Layer
+        :param keywords: Keywords as a dictionary.
+        :type keywords: dict
```
Comment (Gustry) [inline safe/storage/safe_layer.py]: I think it was a mistake in the docstring, isn't it ?

Comment (ismailsunni) [inline safe/storage/safe_layer.py]: Hmm, yes. It seems a bad copy-paste from other method. Thanks :)

**Comment by ismailsunni** [comment]
LGTM

**Pull request merged**